gossip/config.go

```diff
@@ -10,16 +10,24 @@ import (
 )
 
 type (
+	// ProtocolConfig is config for p2p protocol
+	ProtocolConfig struct {
+		// 0/M means "optimize only for throughput", N/0 means "optimize only for latency", N/M is a balanced mode
+
+		LatencyImportance    int
+		ThroughputImportance int
+	}
 	// Config for the gossip service.
 	Config struct {
 		Net     lachesis.Config
 		Emitter EmitterConfig
 		TxPool  evmcore.TxPoolConfig
 		StoreConfig
 
+		TxIndex bool // Whether to disable indexing transactions and receipts or not
```

Comment:
Naming convention of bool: its name should start with Has, Is, Can or Allow, etc.